src/documentation/system_administrators/advanced/quantum_safe_storage/quantum_safe_storage_deploy.md

```diff
@@ -0,0 +1,594 @@
+
+## Deployment Prerequisites
+
+Before proceeding, we assume that you have an activated TFChain account funded with TFT.
+
+- [TFChain accounts](../../../dashboard/wallet_connector.md)
+- [Buy TFT](../../../threefold_token/buy_sell_tft/buy_sell_tft.md)
+- [Send TFT to TFChain](../../../threefold_token/tft_bridges/tfchain_stellar_bridge.md)
+
+## Deploy front end VM
+
+Both full and micro VMs work fine for the QSFS front end.
+
+There are a few considerations to keep in mind for this VM:
+
+- QSFS will consume more CPU and RAM as load increases.
+- 1 vCPU and 2 GB of RAM can work for light loads, but at least an additional vCPU will be helpful for heavier loads.
+- The SSD capacity of the VM is the maximum available front end cache size for QSFS.
+- You should enable Mycelium networking, as this will be required to connect to the zdbs.
+
+You can deploy this VM using the [Dashboard](https://dashboard.grid.tf/#/deploy/virtual-machines/) or via the various other methods described in the ThreeFold Manual.
+
+If you plan to use the front end VM to run tfcmd and deploy the zdbs, then connect to the VM via SSH now and run all the following commands on the VM.
+
+## Deploy Back End Zdbs
+
+Now we will deploy the back end zdbs using tfcmd. To assist with creating multiple deployments efficiently, some short scripts will be shown below. These scripts have been tested in bash and might not work in other shells. At the same time, we will also create our zstor config file, which must contain information about the back ends.
+
+### Prerequisites
+
+We will use `wget`, `jq`, and `openssl`. Make sure that they are properly installed on the system you will be using. On Ubuntu, for example, you can use this command:
+
+```sh
+apt update && apt install -y wget jq openssl
+```
+
+### Set Your EDITOR
+
+In the sections below, we'll be creating a number of config files. Set your preferred editor now to facilitate copying and pasting the commands. For example:
+
+```sh
+EDITOR=nano
+```
+
+### Install Tfcmd
+
+Here we will fetch the `tfcmd` binary and install it locally. It is released as part of `tfgrid-sdk-go`. You can find the latest releases [here](https://github.com/threefoldtech/tfgrid-sdk-go/releases). Choose the correct download link for your platform.
+
+For example, to install version 15.10 on a x64 Linux machine, you would run the following:
+
+```
+wget -O tfgrid-sdk-go.tar.gz https://github.com/threefoldtech/tfgrid-sdk-go/releases/download/v0.15.10/tfgrid-sdk-go_Linux_x86_64.tar.gz
+tar -C /usr/local/bin -xf tfgrid-sdk-go.tar.gz tfcmd
+chmod +x /usr/local/bin/tfcmd
+rm tfgrid-sdk-go.tar.gz
+```
+
+To test that it worked, you can use the following command:
+
+```sh
+tfcmd version
```

Comment:
You will still need to login to the TFCMD first.

Reply:
Good point. I have replaced this check with instructions to login.